hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/impl/ContainerDataYaml.java

```diff
@@ -71,13 +73,19 @@ private ContainerDataYaml() {
    * Creates a .container file in yaml format.
    */
   public static void createContainerFile(ContainerData containerData, File containerFile) throws IOException {
+    if (disableWrite) {
```

Comment:
Is it possible that after disableWrite is from true to false , there are new updates of yaml files?

Reply:
This is a precautionary code, in theory, this branch should never be reached, the cache file will be generated after other Datanode Servers are stop, there should not be any Yaml/Container update operation in this time.